lib/src/platform_icons.dart

```diff
@@ -539,6 +539,12 @@ class PlatformIcons {
       : IconData(0xf2fc,
           fontFamily: 'CupertinoIcons', fontPackage: 'cupertino_icons');
 
+  /// Icons: Icons.star_outline : CupertinoIcons.(custom)
+  IconData get star_outline => isMaterial(context)
```

Comment:
Please name this so it matches the pattern of the other icons

```suggestion
  IconData get starOutline => isMaterial(context)
```